src/components/web-monitoring-ui.jsx

```diff
@@ -65,14 +68,29 @@ export default class WebMonitoringUi extends React.Component {
   logOut () {
     api.logOut();
     this.setState({user: api.userData});
-    this.loadPages('pages');
+    this.loadPages();
   }
 
-  search (query) {
-    this.setState({search: query});
+  urlSearch (urlQuery) {
+    this.setState({urlQuery});
     this.loadPages();
   }
 
+  dateSearch(startDateQuery, endDateQuery) {
+    this.setState({startDateQuery, endDateQuery}, function () {
+      let captureTime = null;
+      if (this.state.startDateQuery || this.state.endDateQuery) {
+        captureTime = {
+          capture_time: {
+            startDateQuery: this.state.startDateQuery,
+            endDateQuery: this.state.endDateQuery
+          }
+        };
+      }
+      this.loadPages(captureTime);
```

Comment:
I don’t think `captureTime` should be special here. `loadPages()` needs to pull it out of the state like it does everything else, and not receive it as a special argument.